webapp/src/ts/modals/edit-user/update-password.component.ts

```diff
@@ -127,7 +128,7 @@ export class UpdatePasswordComponent {
   private async validateConfirmPasswordMatches() {
     if (this.editUserModel.password !== this.editUserModel.passwordConfirm) {
       const value = await this.translateService.get('Passwords must match');
-      await this.setError(ErrorType.PASSWORD, value);
+      this.setError(ErrorType.PASSWORD, value);
```

Comment:
Thank you!

I feel like eslint should be able to detect this?

Reply:
I feel like that as well.